rt_dashboard/javascript/src/components/dashboard/dashboard.js

```diff
@@ -1,13 +1,17 @@
 import templateHtml from './dashboard.html'
 import { loadTemplate } from '../../utils/dom'
 
+const REFRESH_INTERVAL = 3000
```

Comment:
In #41 this interval is provided in the script tag as [poll](https://github.com/djangsters/rt-dashboard/pull/41/files#diff-6f92862c477a3dfc59e25cf4ac3aacffR8). Can use that and fallback